.github/workflows/node.js.yml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on:
+  push
+  pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
im not sure but this package can only working on 14.x or upper

Reply:
Hmm, we can remove other and just use 14.x 15.x 16.x